src/Microsoft.AspNet.Razor/RazorResources.resx

```diff
@@ -376,7 +376,7 @@ Instead, wrap the contents of the block in "{{}}":
     <value>The tag helper '{0}' must not have C# in the element's attribute declaration area.</value>
   </data>
   <data name="TagHelpers_TagHelperCodeGeneartorMustBeAssociatedWithATagHelperBlock" xml:space="preserve">
-    <value>A TagHelperCodeGenerator must only be used with TagHelperBlocks.</value>
+    <value>A TagHelperChunkGenerator must only be used with TagHelperBlocks.</value>
```

Comment:
Hmm, the names `TagHelperChunkGenerator` and `TagHelperBlock` should be passed into this resource. Not that the message is all that useful externally -- sounds like a `Debug.Assert()` message.